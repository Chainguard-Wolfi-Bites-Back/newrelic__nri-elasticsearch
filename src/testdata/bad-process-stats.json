{
    "process": {
        "missingjsonkey": 1000,
        "id": "15171",
        "mlockall": "false"
    }
}