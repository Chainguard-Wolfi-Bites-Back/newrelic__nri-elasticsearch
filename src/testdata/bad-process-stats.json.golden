{"name":"Test","protocol_version":"3","integration_version":"0.0.1","data":[{"metrics":[],"inventory":{"config/process/id":{"value":0},"config/process/mlockall":{"value":false}},"events":[]}]}